allennlp/tests/interpret/attack.py

```diff
@@ -0,0 +1,36 @@
+# pylint: disable=no-self-use,invalid-name
+from pytest import approx
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.models.archival import load_archive
+from allennlp.predictors import Predictor
+from allennlp.interpret.attackers import InputReduction, Hotflip
+
+class TestAttack(AllenNlpTestCase):
+    def test_attackers(self):
+        inputs = {
+            "premise": "I always write unit tests for my code.",
+            "hypothesis": "One time I didn't write any unit tests for my code."
+        }
+
+        archive = load_archive(self.FIXTURES_ROOT / 'decomposable_attention' / 'serialization' / 'model.tar.gz')
+        predictor = Predictor.from_archive(archive, 'textual-entailment')
+        result = predictor.predict_json(inputs)
+
+        # Hotflip
+        hotflipper = Hotflip(predictor)
```

Comment:
Looks like this is a test for `HotFlip`, not for `Attacker`.  For ease of finding and browsing through tests, it's better to separate these out to specific files.  This looks like a sufficient test for the hotflip attacker (and similarly below), it should just live in `hotflip_test.py`, or whatever the right path is.